source/maintainer-guide/installing/installing-from-docker.rst

```diff
@@ -0,0 +1,47 @@
+.. _installing-from-docker:
+
+Installing from docker
+######################
+
+
+Overview
+--------
+
+Docker is a popular virtualisation technology in the web hosting domain. 
+Docker can also be used to try out new software without the need to install various dependencies.
+The GeoNetwork community maintains 'Docker images' ready to run in your Docker environment. 
+These images are available at 'Docker hub': https://hub.docker.xom/_/geonetwork.
+Docker desktop includes a tool called Kitematic, which can be used to download and run docker images from Docker hub.
+This article guides you in getting started with GeoNetwork by using Kitematic and Docker.
+
+Before you Start
+----------------
+
+Install the latest version of Docker desktop including Kitematic from https://www.docker.com/products/docker-desktop.
+
+Steps to get started with GeoNetwork
+------------------------------------
+
+Start the kitematic application. On some operating systems Kitematic will now start a 'Docker VM', in which the docker images will run.
+
+Use the search bar to search in docker hub with a keyword 'GeoNetwork'. Click the 'create' button on the official GeoNetwork card.
+
+.. figure:: img/kitematic-search.png
+
+Kitematic will download the docker image and start it in the docker environment. This will take a couple of minutes, depending on your internet bandwith.
+
+An instance of an image running in the docker environment is called a 'container'. Take a look at the container properties and verify on which port the container is running.
```

Comment:
You may wish to have discussion and definitions above in the description, they get in the way of step by step instructions (asking folks to understand new terms, while performing actions it tough).

```
#. Take a look at your new GeoNetwork container properties and verify address it has been assigned.

#. Use your browser to open GeoNetwork: ``http://<IP address>:8080/geonetwork/``

     .. figure:: img/docker.png
```